synapse/cortex.py

```diff
@@ -493,6 +493,21 @@ def setFeedOffs(self, iden, offs):
         async for mesg in self.cell.streamstorm(text, opts, user=self.user):
             yield mesg
 
+    async def syncLayerSplices(self, iden, offs):
+        '''
+        Yield (indx, mesg) splices for the given layer beginning at offset.
+
+        Once caught up, this API will begin yielding splices in real-time.
+        The generator will only terminate on network disconnect or if the
+        consumer falls behind the max window size of 10,000 splice messages.
+        '''
+        if not self.user.admin:
```

Comment:
I'd like to avoid the need for complex configuration, but yea we can add a special perms check here ( which if you just wanna use an admin account will pass anyway )

Reply:
1. We have a decorator for requiring explicit admin rights
2. A safer scenario is that you may want a service account to do the syncing.  A perm like `layer:sync` would probably make sense, in line with the other remote layer api permissions.